Source/WebCore/page/DebugPageOverlays.cpp

```diff
@@ -177,15 +177,15 @@ static void drawRightAlignedText(const String& text, GraphicsContext& context, c
 
 void NonFastScrollableRegionOverlay::drawRect(PageOverlay& pageOverlay, GraphicsContext& context, const IntRect&)
 {
-    static constexpr std::pair<ComparableASCIILiteral, SRGBA<uint8_t>> colorMappings[] = {
-        { "mousedown", { 80, 245, 80, 50 } },
-        { "mousemove", { 245, 245, 80, 50 } },
-        { "mouseup", { 80, 245, 176, 50 } },
-        { "touchend", { 191, 63, 127, 50 } },
-        { "touchforcechange", { 63, 63, 191, 50 } },
-        { "touchmove", { 80, 204, 245, 50 } },
-        { "touchstart", { 191, 191, 63, 50 } },
-        { "wheel", { 255, 128, 0, 50 } },
+    static constexpr std::pair<EventTrackingRegions::Event, SRGBA<uint8_t>> colorMappings[] = {
+        { EventTrackingRegions::Event::MousedownEvent, { 80, 245, 80, 50 } },
```

Comment:
I think we should capitalize the start of every word instead e.g. MouseDownEvent.  Ditto for other events below.

Reply:
Ah, no. Because this is DOM's `"mousedown"` event. So we should align them with DOM's definition.